src/components/Text/docs/TextPropsTable.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { PropsTable } from '../../../docs/PropsTable';
+
+export const TextPropsTable = () => {
+    const props = [
```

Comment:
Should we for now include the `weak` prop in the props table? I don't have a strong opinion, but it would be consistent and allow users to find the things they're currently already using.

Reply:
I added the prop to the table with the deprecated label